### DISCUSSION TITLE
Dual/Triple NIC on a VM

### DISCUSSION THREAD

**Discussion by taslabs-net** [tteck/Proxmox#3597] [Help Needed]
What is the proper way to give a VM more than one network connection? For instance with Zoraxy, NPM, etc I have the ability to load balance two networks to the proxies.  When I install, then go back and add another NIC, the VM/LXC's aren't reachable anymore.  

Best practice? I appreciate the insights :)

**Comment by P1ctur3** [comment]
adding 2 NICs to my VM or LXC never get into trouble.

**Reply by taslabs-net** [reply]
The node itself has 3 networks, with 1 default gateway.  Do you create the VM/LXC the same? Do you only assign IPs and no default gateway to the vm/lxc? what's the right answer?

PVE Node
![image](https://github.com/user-attachments/assets/662c560a-31e8-439b-bbdc-63fa09480780)

VM/LXC 
![image](https://github.com/user-attachments/assets/08961ead-f1cc-4cd0-8963-bcf54a1f2f00)
